docs/autoscaling.md

```diff
@@ -60,21 +60,21 @@ The latest stable version AutoScaling Engine can be cloned from this [repository
 git clone https://github.com/openbaton/autoscaling.git
 ```
 
-Once this is done, go inside the cloned folder and make use of the provided script to compile the project as done below:
+Once this is done, go inside the cloned folder and make use of the provided script to compile the project and start as done below:
 
 ```bash
-./autoscaling-engine.sh compile
+./autoscaling-engine.sh compile start
 ```
 
 # Manual configuration of the AutoScaling Engine
 
 This chapter describes what needs to be done before starting the AutoScaling Engine. This includes the configuration file and properties, and also how to make use of monitoring plugin.
 
 ## Configuration file
```

Comment:
@mpauls @flaviomu please double check which one is the right file name to be used.